frontend/src/pages/article/shared-article-page.tsx

```diff
@@ -62,6 +62,8 @@ const SharedArticlePage: React.FC = () => {
                   publishedAt={article.publishedAt ?? ''}
                   genre={article.genre ?? ''}
                   avatarUrl={article.author.avatarUrl}
+                  authorFollowers={article.author.followersCount}
+                  isShared={true}
```

Comment:
Could be simplified
```suggestion
                  isShared
```

Reply:
[1138609](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/280/commits/1138609f69c165918af13092d761523d0baed61c)